Adding token balances limitation info in swagger docs

mgoelswirlds opened the pull request:
**Description**:
As noted in [HIP-367](https://hips.hedera.com/hip/hip-367#mirror-node) the balances endpoint is limited to at most 50 token balances per account. Adding this information to mirror node swagger docs.

This PR modifies :  
* Add token balances limitation to swagger docs for /accounts and /balances endpoint.

**Related issue(s)**:

Fixes #5331

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

steven-sheehy reviewed (changes requested):
Need to put other endpoint limitations from HIP-367 as noted in my issue comment.

mgoelswirlds commented:
> limitations

@steven-sheehy Do you mean we should also mention why we have this limitation. and the alternative people can use?

steven-sheehy commented:
No, but now that you bring it up it might be good to discuss the alternative API to use. Don't have to go into why we have limitation. Just say as `outlined in HIP-367`.

But my comment was about accounts list and get missing a similar treatment.

mgoelswirlds commented:
I did missed account get , which I have added now. But for account list , I added the description near the balance param, as I thought that is more relevant.

steven-sheehy reviewed (approved):
LGTM

edwin-greene reviewed (approved):
Looks good
